scripts/dojson

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
I do not create this but add this command to the `./scripts/setup` as a comment. Same as loans.

Reply:
We dont like to recreate every time the 10k.json file